feature/amazon/src/test/java/com/revenuecat/purchases/amazon/ReceiptToStoreTransactionTest.kt

```diff
@@ -22,7 +21,7 @@ class ReceiptToStoreTransactionTest {
         val receipt = dummyReceipt()
 
         val storeTransaction = receipt.toStoreTransaction(
-            sku = "sku",
+            productId = "sku",
```

Comment:
Should we rename this value in all these tests from `sku` to `productId`? Or maybe even extract to a constant. It shouldn't block this though.

Reply:
actually in amazon they are called "skus" 🤷‍♀️